tests/tools/test_release_version.py

```
@@ -0,0 +1,231 @@
+
+        assert result.returncode == 0, result.stderr
+        assert "Next Safe-Synthesizer post-release" in result.stdout
+        assert "Next post-release tag: v1.6.0.post1" in result.stdout
+        assert "No tag was created, deleted, or pushed." in result.stdout
+        assert self._git("tag", "--list") == tags_before
+
+    def test_post_release_without_matching_stable_tag_is_refused(self) -> None:
+        self._tag("v1.5.0")
+        self._tag("v1.6.0.post1")
+
+        with pytest.raises(self.release_version.ReleaseVersionError, match="missing stable tag v1.6.0"):
```

Ruff (0.15.21)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)